### ISSUE TITLE
use openNetVM quesstion?

### ISSUE THREAD

**Issue by awzhgw** [mtcp-stack/mtcp#273]
I have a distributed storage software that wants to use mtcp. I need to start 2 processes (metanode and datanode) on each machine. I have 8 machines. Each machine needs to start metanode and datanode, and each process needs Communicate with each other, then I use openNetVM. Can I meet my needs?

**Comment by a-jamshed** [comment]
Yes. Please contact OpenNetVM authors for details (e.g. @koolzz).

**Comment by koolzz** [comment]
Yeah this can be implemented using openNetVM. You can run 2 NFs(metanode and datanode) on the same host and communicate over messages. Feel free to post an issue with specifics and we'll be able to help you out https://github.com/sdnfv/openNetVM
